modules/sriov-network-metrics-exporter.adoc

```diff
@@ -0,0 +1,71 @@
+//
+// * networking/hardware_networks/configuring-sriov-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="sriov-network-metrics-exporter_{context}"]
+= About the SR-IOV network metrics exporter
+
+The Single Root I/O Virtualization (SR-IOV) network metrics exporter reads the metrics for SR-IOV virtual functions (VFs) and exposes these VF metrics in Prometheus format. When the SR-IOV network metrics exporter is enabled, you can query the SR-IOV VF metrics by using the {product-title} web console to monitor the networking activity of the SR-IOV pods.
+
+When you query the SR-IOV VF metrics by using the web console, the SR-IOV network metrics exporter fetches and returns the VF network statistics along with the name and namespace of the pod that the VF is attached to.
+
+The SR-IOV VF metrics that the metrics exporter reads and exposes in Prometheus format are described in the following table:
+
+.SR-IOV VF metrics
+[%autowidth,options="header"]
+|====
+|Metric| Description |Example PromQL query to examine the VF metric
+
+|`sriov_vf_rx_bytes` |Received bytes per virtual function. |`sriov_vf_rx_bytes * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_tx_bytes` |Transmitted bytes per virtual function. |`sriov_vf_tx_bytes * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_rx_packets` |Received packets per virtual function. |`sriov_vf_rx_packets * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_tx_packets` |Transmitted packets per virtual function. |`sriov_vf_tx_packets * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_rx_dropped` |Dropped packets upon receipt per virtual function. |`sriov_vf_rx_dropped * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_tx_dropped` |Dropped packets during transmission per virtual function. |`sriov_vf_tx_dropped * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_rx_multicast` |Received multicast packets per virtual function. |`sriov_vf_rx_multicast * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_vf_rx_broadcast` |Received broadcast packets per virtual function. |`sriov_vf_rx_broadcast * on (pciAddr) group_left(pod,namespace,dev_type) sriov_kubepoddevice`
+|`sriov_kubepoddevice` |Virtual functions linked to active pods. |-
+
+|====
+
+You can also combine these queries with the kube-state-metrics to get more information about the SR-IOV pods. For example, you can use the following query to get the VF network statistics along with the application name from the standard Kubernetes pod label:
+[source,bash]
+----
+(sriov_vf_tx_packets * on (pciAddr)  group_left(pod,namespace)  sriov_kubepoddevice) * on (pod,namespace) group_left (label_app_kubernetes_io_name) kube_pod_labels
+----
+
+[NOTE]
+====
+To view the network statistics of a VF by using the {oc-first}, run the following command:
```

Comment:
If we go down this way, we have to explain the user to open a shell in the worker node, that is not related to the `oc` client. 
This is not how the feature is supposed to be used, it's an information for the user to know where these numbers come from, as he might be used to kernel device statistics.

That said, I'm in favor of removing this note: the introduction you wrote at the beginning of this chapter is clear enough.

If you prefer to keep it, please specify that the user has to run the `ip -s link ...` command on the worker node. Your call

Reply:
@zeeke 
I agree with you!
This note is just additional info. Also, the metrics have been described in the table with example so I think we can skip the note.

Updated as per the suggestion.